sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/model/ValueMappingModel.java

```diff
@@ -0,0 +1,47 @@
+package org.sdase.commons.keymgmt.model;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import java.util.*;
+
+@SuppressWarnings("unused")
+public class ValueMappingModel {
+
+  private final Map<String, String> apiToImplBidirectional = new HashMap<>();
+  private final Map<String, String> implToApiBidirectional = new HashMap<>();
+
+  private final Map<String, String> implToApi = new HashMap<>();
+
+  private final Map<String, String> apiToImpl = new HashMap<>();
+
+  @JsonSetter("api_to_impl_bidirectional")
```

Comment:
In all (most?) situations we use camel case in sda-commons. Is there a reason to use snake case here?

Reply:
I used snake case here since I think that it is better to read in the mapping files.

Reply:
switched to camel case